stages/stage_0.py

```diff
@@ -9,7 +9,7 @@
 def run_program(
     program,
     args,
-    quote_kw=KEYWORD_TO_ATOM["q"],
+    quote_kw=QUOTE_ATOM,
     apply_kw=KEYWORD_TO_ATOM["a"],
```

Comment:
it still bothers be that the names of these parameters are "keywords" when they are "atoms"

Reply:
Me too - I'll get to stuff like that after landing the quote change.